.ci/magician/utility/utils.go

```diff
@@ -219,3 +230,9 @@ func ReadFromJson(data interface{}, path string) error {
 	}
 	return nil
 }
+
+func ParseInt(s string) int {
+	var num int
+	fmt.Sscanf(s, "%d", &num)
+	return num
+}
```

Comment:
This doesn't seem to be used? Also, I think it duplicates https://pkg.go.dev/strconv#Itoa

```suggestion
```

Reply:
Ah this snuck in from earlier, removed.